### PR TITLE
feat: vertical tab with line

### PR DESCRIPTION
**Pull request by anshul-pinto0410**
## Checklist

- [ ] PR is up-to-date with a description of changes and screenshots (if applicable).
- [ ] All files are lint-free.
- [ ] Followed Semantic PR guidelines.
- [ ] Tested locally for regressions.

### CODE REVIEW

File: docs/sidebars.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ module.exports = {
         label: "Tabs",
         items: [
           "tabs",
+          "vertical-tabs-w-line",
```
Comment (pritam248) [inline docs/sidebars.js]:
```suggestion
          "line-tab",
```
i guess we wont be having line-tab horizontal 😄

File: src/components/Tabs/tabs.css [hunk 2]
```diff
@@ -32,20 +32,11 @@
     }
     .lil-tabs > .lil-tab {
         min-width: 150px;
-        @apply lil-text-base lil-mr-2 lil-rounded-md;
-    }
-    .lil-tabs .lil-tab-active {
-        @apply lil-bg-blue-350;
-    }
-    .lil-tab-active .lil-tab-heading-text {
-        @apply lil-text-white;
```
Comment (pritam248) [inline src/components/Tabs/tabs.css:41]: is it not causing regression ?

File: src/components/Tabs/verticalTabWLine.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.lil-vertical-tabs{
+    @apply lil-list-none lil-inline-block lil-overflow-x-auto;
+}
+.lil-vertical-tabs .lil-vertical-tab{
+    min-height:100px;
+    @apply lil-opacity-50 lil-flex lil-whitespace-normal lil-cursor-pointer lil-outline-none;
+}
+.lil-vertical-tab .lil-vertical-tab-container{
+    @apply lil-flex lil-items-center lil-w-full  lil-border-l-xl lil-border-font-200 lil-flex-wrap;
```
Comment (pritam248) [inline src/components/Tabs/verticalTabWLine.css]: `lil-items-center` let this be decided by consumer

File: src/components/Tabs/verticalTabWLine.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+.lil-vertical-tab-active .lil-vertical-tab-content{
+    @apply lil-border-opacity-50 lil-w-11/12;
+}
+
+.lil-vertical-tab-active .lil-vertical-tab-arrow{
+    @apply lil-hidden;
+}
+
+.lil-vertical-tab-active .lil-vertical-tab-writeup{
+    @apply lil-flex lil-text-base lil-text-font-200 lil-py-3 lil-visible;
```
Comment (pritam248) [inline src/components/Tabs/verticalTabWLine.css]: `lil-visible` this is for ?

File: src/variables.css [hunk 5]
```diff
@@ -12,6 +12,7 @@
     --grey-500: #8B8F99;
     --grey-550: #686D77;
     --grey-600: #23282E;
+    --blue-350: #1678FB;
```
Comment (pritam248) [inline src/variables.css]: plz organize in ascending order & make sure the shade matches number given

File: tailwind.config.js [hunk 6]
```diff
@@ -9,7 +9,7 @@ module.exports = {
         200: "#8ABBFD",
         250: "#7FB7FE",
         300: "#2F89FC",
-        350: "#1678FB",
+        350: "var(--blue-350, #1678FB)",
```
Comment (pritam248) [inline tailwind.config.js:12]: 👍

File: docs/docs/vertical-tab-w-line.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+                        </div>
+                        <div class="lil-w-5/6">
+                            <div class="lil-vertical-tab-heading">Import </div>
+                        </div>
+                        <div class="lil-vertical-tab-writeup">
+                            File in under 30 seconds with power workflows such as NIL
+                            filing and auto-filed 3B and 9 Forms.
+                        </div>
+                    </div>
+                    <div class="lil-vertical-tab-arrow">&#8594;</div>
```
Comment (pritam248) [inline docs/docs/vertical-tab-w-line.md]: span may be

File: docs/docs/vertical-tab-w-line.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+---
+
+``` html live
+<>
+<div class="lil-flex"> 
+    <div class="lil-w-2/5 sm:lil-hidden">
+        <ul class="lil-vertical-tabs">
+            <li class="lil-vertical-tab lil-vertical-tab-active" onclick="verticalTabsWLine(event,'tab1')">
+                <a class="lil-vertical-tab-container">
+                    <div class="lil-vertical-tab-line"></div>
```
Comment (pritam248) [inline docs/docs/vertical-tab-w-line.md]: y cant we left border for tab ? 🤔

File: src/components/Tabs/tabs.css (hunk 2 above)
Comment (anshul-pinto0410) [reply src/components/Tabs/tabs.css:41]: Only changed active styling for small screen here, checked for both home & individuals

File: src/components/Tabs/verticalTabWLine.css (hunk 4 above)
Comment (anshul-pinto0410) [reply src/components/Tabs/verticalTabWLine.css]: in acitve state of tab, the description is visible in CTA where as it is hidden in the other case

File: docs/docs/vertical-tab-w-line.md (hunk 8 above)
Comment (anshul-pinto0410) [reply docs/docs/vertical-tab-w-line.md]: Tried this, actually there are two left borders of grey and blue for active tab, the blue one is not of full height so we have a separate element of border height

Comment (anshul-pinto0410) [reply docs/docs/vertical-tab-w-line.md]: there is also a part where there is no bottom border, hence it's more straight forward using another element, tried using margins to simulate this but didn't work properly, if I but the bottom border for the parent we won't be able to have that space in between, and if bottom border is for current element then the height of left border becomes full

Comment (anshul-pinto0410) [reply docs/docs/vertical-tab-w-line.md]: Screenshot from 2020-11-10 10-52-56

Comment (pritam248) [reply docs/docs/vertical-tab-w-line.md]: > there is also a part where there is no bottom border

👀

**Review by pritam248 — APPROVED** [review]

**Pull request merged**